point.py

```diff
@@ -0,0 +1,16 @@
+from . import utils
+
+class Foo(object):
```

Comment:
Maybe name it `Point`?  Easier for the next developer to know that `Foo` is actually a 2D point object.  Just a style comment, not a correctness issue.